content-src/components/DiscoveryStreamComponents/DSTextPromo/DSTextPromo.jsx

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { SafeAnchor } from "../SafeAnchor/SafeAnchor";
+
+export class DSTextPromo extends React.PureComponent {
+  render() {
+    return (
+      <div className="ds-text-promo">
+        <img src={this.props.image} alt={this.props.alt_text} />
+        <div className="text">
+          <header>
```

Comment:
nit: as per accessibility we should using headings in favor of header which should be only one in a section https://www.w3.org/WAI/tutorials/page-structure/regions/